DiskIOMeter.c

```diff
@@ -136,32 +208,156 @@ static void DiskIOMeter_display(ATTR_UNUSED const Object* cast, RichString* out)
    char buffer[16];
 
    int color = cached_utilisation_diff > 40.0 ? METER_VALUE_NOTICE : METER_VALUE;
-   int len = xSnprintf(buffer, sizeof(buffer), "%.1f%%", cached_utilisation_diff);
+   int len = xSnprintf(buffer, sizeof(buffer), "%4.1f%%", cached_utilisation_diff);
```

Comment:
This looks inconsistent with signle-digit percentages having an additional blank. Just use minimal spacing.
```suggestion
   int len = xSnprintf(buffer, sizeof(buffer), "%.1f%%", cached_utilisation_diff);
```